OpenSim/Simulation/Model/OffsetFrame.h

```diff
@@ -90,7 +95,7 @@ class  OffsetFrame : public C {
     offset property of this OffsetFrame.
      
     @param[in] parent   The parent reference frame.
-    @param[in] offset The offset transform between this frame and its parent
+    @param[in] offset   The offset transform between this frame and its parent
     */
     OffsetFrame(const C& parent, const SimTK::Transform& offset);
```

Comment:
> it seems like with C++11 we can replace such const pass-by-reference's with pass-by-value

This makes sense for things that get copied, but not for intentional references (which is what I think this one is). Using pass by value would cause a copy to occur whenever an lvalue is passed. Tighter control can be had by overloading a reference (&) and rvalue reference (&&) version of the method.